### PR TITLE
Issue325 object length linter

### PR DESCRIPTION
**Pull request by DragosMG**
Made the default value of the length argument 30 (consistent across use, test, etc).  This should close issue #325.

### CODE REVIEW

File: R/zzz.R [hunk 1]
```diff
@@ -94,7 +94,7 @@ default_linters <- with_defaults(default = list(),
   infix_spaces_linter,
   line_length_linter(80),
   no_tab_linter,
-  object_length_linter(30),
+  object_length_linter,
```
[the remark this answers is not in the document]
Comment (DragosMG) [reply R/zzz.R]: Cool. Changed

**Comment by jimhester** [comment]
Thanks!

Could you add a note to NEWS.md with this change; referencing the issue number and your GitHub username, then I can merge this.

**Comment by DragosMG** [comment]
Note added to NEWS.md. Thanks!

**Comment by jimhester** [comment]
Thanks!

**Pull request merged**